core/include/tee/tee_cryp_provider.h

```diff
@@ -181,6 +181,20 @@ struct dh_keypair {
 	uint32_t xbits;		/* Number of bits in the private key */
 };
 
+/* ECC (optional) */
```

Comment:
True, ECC is optional in the spec, but this comment isn't needed here or elsewhere. Unless it's needed to explain some clever idfef or something.